woocommerce-payments.php

```diff
@@ -106,7 +106,7 @@ function wcpay_init() {
 	 * Initialise subscriptions-core if WC Subscriptions (the plugin) isn't loaded
 	 */
 	function wcpay_init_subscriptions_core() {
-		if ( ! WC_Payments_Features::is_wcpay_subscriptions_enabled() ) {
+		if ( WC_Payments_Features::is_wcpay_subscriptions_enabled() ) {
```

Comment:
@mattallan was this change intentional? With this change, I'm no longer able to access subscriptions functionality

Reply:
Oh my! No, it wasn't intentional to break everything haha 🙈 Thanks for catching this!!

Reply:
Fixed in 8d45beb